packages/edit-post/src/components/visual-editor/index.js

```diff
@@ -57,13 +63,18 @@ export default function VisualEditor() {
 	useClipboardHandler( ref );
 	useTypingObserver( ref );
 	useCanvasClickRedirect( ref );
+	const darkEditorRef = useDarkEditorStyle();
+	const mergedRefs = useCallback( mergeRefs( [ ref, darkEditorRef ] ), [
```

Comment:
I believe we might have a hook in the  works to replace this with `useMergeRefs`. Is this ready yet @ellatrix ?